dotcom-rendering/src/web/components/Card/Card.stories.tsx

```diff
@@ -30,6 +30,7 @@ const basicCardProps: CardProps = {
 		'https://media.guim.co.uk/6537e163c9164d25ec6102641f6a04fa5ba76560/0_210_5472_3283/master/5472.jpg',
 	imagePosition: 'top',
 	showAge: true,
+	isExternalLink: false,
```

Comment:
Do we have a story anywhere for either Card or CardHeadline etc that has an external link variation? Feels like it could be useful coverage/documentation?

Reply:
Good point, not at the moment as far as I can tell! I'll add one 👍